src/main/java/nl/hsac/fitnesse/fixture/slim/JsonHttpTest.java

```diff
@@ -66,6 +66,17 @@ public String allJsonPathMatches(String expr) {
         return result;
     }
 
+    /**
+     * Update a value in a the response by supplied jsonPath
+     * @param path the jsonPath to locate the key whose value needs changing
+     * @param value the new value to set
+     */
+    public void setJsonPathTo(String path, String value){
+        String jsonStr = getResponse().getResponse();
+        String jsonPath = getPathExpr(path);
+        getResponse().setResponse(pathHelper.updateJsonPathWithValue(jsonStr, jsonPath, value));
```

Comment:
Please make a local variable with the new response, before setting it. That makes debugging just so much easier...